vite.config.ts

```diff
@@ -21,6 +21,9 @@ export default defineConfig({
         "./app/routes/**/*",
       ],
     },
+    hmr: {
```

Comment:
Not sure what this is. Can you give me some explanation

Reply:
When an error occurs, Vite typically displays an overlay in the browser to inform you of the error. This was added to remove the error coming up on the browser while local development. This can be removed now!